client/src/app/home.js

```diff
@@ -272,483 +272,23 @@ const Home = observer((props) => {
               style={{ maxWidth: 1000, marginBottom: 1 }}
             >
               <center className="waifu-card-box-center">
-                <div className="presale-title">Presale</div>

```

Comment:
should this buy button die? https://github.com/chase-manning/waifusion-dungeon/pull/19/files#diff-e7ee87608ae0e647852df7423e9cef535885d4ade3769fcc7d4eb138faa6064aR244-R254

What about the rest of the buy logic/modal in this file? At the end of the only way to interact with the contract is via the dungeon, right?

Reply:
link to the lines of code doesn't seem to be working but its this button from lines 244-253 in this file 
![image](https://user-images.githubusercontent.com/16842072/110195652-e9572d80-7dfb-11eb-90d5-890e9c6003bf.png)